site/_js/slider.js

```diff
@@ -0,0 +1,183 @@
+import {gsap} from 'gsap';
+import {ScrollTrigger} from 'gsap/ScrollTrigger';
+import {ScrollToPlugin} from 'gsap/ScrollToPlugin';
+
+gsap.registerPlugin(ScrollTrigger, ScrollToPlugin);
+
+const cssClasses = {
+  NAVIGATION_WRAPPER: 'navigation-wrapper',
+  FIXED_NAVIGATION_WRAPPER: 'fixed-navigation-wrapper',
+  YEAR_NAVIGATION: 'year-navigation',
+  FIXED_YEAR_NAVIGATION: 'fixed-year-navigation',
+  MENU: 'menu',
+  FIXED_MENU: 'fixed-menu',
+  MENU_ITEM: 'menuitem',
+  MENU_ITEM_LINK: 'sectionLink',
+  CHROME_LOGO: 'chrome-logo',
+  YEAR_SECTION: 'inner-card',
+  HIDE_NAVIGATION: 'hide-navigation',
+};
+
+class Slider {
```

Comment:
I have not been able to test out this code because the `gasp` library isn't being imported properly. (Please see my earlier comment about loading it in from the CDN, and keeping it out of our main bundle.)

 I would very much like to spend some time evaluating the live performance of this implementation prior to merging, as I see some potential trouble spots. I don't want to flag anything without actually profiling the code in a real browser, though.

Reply:
Done